light-curve/tests/light_curve_py/features/test_roms.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from light_curve.light_curve_py import Roms
+
+
+def test_roms_1():
+    feature = Roms()
+    n = 100
+    t = np.arange(n)
+    m = np.ones_like(n)
```

Comment:
This would make a single value, not an array. I think you meant it to be `np.ones(n)` or `np.ones_like(t)`